requirements.txt

```diff
@@ -6,11 +6,17 @@ charset-normalizer==2.0.12
 click==8.1.3
 enlighten==1.10.1
 fastapi==0.78.0
+fugashi[unidic-lite]==1.1.2
 h11==0.13.0
+hanziconv==0.3.2
 httptools==0.4.0
 idna==3.3
+jieba==0.42.1
+laserembeddings[ja,zh]==1.1.2
```

Comment:
Is this just for Chinese and Japanese? We should have a mechanism to install laser for whatever the user needs it at the moment? Maybe a separate installation script as opposed to `requirements.txt`?

Reply:
`ja` and `zh` are extras, these languages are not supported by default. AFAIK this is a way to get complete language support.

There is an additional required step, I'm not sure where to put it (probably in the README, at least for now?):

```
python -m laserembeddings download-models
```

Reply:
Yeah, adding that to a section of the readme for now sounds like the better option. I was discussing with Nick how this would work in the future, e.g. make the filter step download any necessary dependencies when it needs them for the first time. But that would not work on e.g. a compute node without internet access.

Reply:
Thanks, done. As far as I can tell, the filter is working, removing Draft status.